### ISSUE TITLE
[YSQL] Stored generated column is not supported

### ISSUE THREAD

**Issue by tedyu** [yugabyte/yugabyte-db#10695]
Jira Link: [DB-1115](https://yugabyte.atlassian.net/browse/DB-1115)
### Description

```
yugabyte=# CREATE TABLE gtest0 (a int PRIMARY KEY, b int GENERATED ALWAYS AS (55) STORED);
ERROR:  syntax error at or near "("
LINE 1: ...st0 (a int PRIMARY KEY, b int GENERATED ALWAYS AS (55) STORE...
                                                             ^
```

**Comment by saurik** [comment]
Apparently this was a PostgreSQL 12 feature (and Yugabyte is based on PostgreSQL 11.2).

**Comment by TroyKomodo** [comment]
Would be nice if this ticket made it into a milestone.

**Comment by smiziara** [comment]
+1, cannot use some packages like zitadel, due to lack of this feature.

**Comment by sushantrmishra** [comment]
Fixed as part of https://github.com/yugabyte/yugabyte-db/issues/24291 . This should be available in 2025.1 release of YugabyteDB in which YugabyteDB is update to PG15 compatible. 

duplicate of https://github.com/yugabyte/yugabyte-db/issues/24291
